flexmeasures/data/migrations/versions/830e72a8b218_migrate_sensor_relationships_for_power_price_weather.py

```diff
@@ -0,0 +1,132 @@
+"""Migrate sensor relationships for Power/Price/Weather
+
+Revision ID: 830e72a8b218
+Revises: 6cf5b241b85f
+Create Date: 2021-12-02 14:58:06.581092
+
+"""
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = "830e72a8b218"
+down_revision = "6cf5b241b85f"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+
+    # Migrate Power/Asset relationship to Power/Sensor relationship
+    op.drop_constraint("power_asset_id_asset_fkey", "power", type_="foreignkey")
+    op.drop_index("power_asset_id_idx", table_name="power")
+    op.alter_column("power", "asset_id", new_column_name="sensor_id")
+    op.create_index(op.f("power_sensor_id_idx"), "power", ["sensor_id"], unique=False)
+    op.create_foreign_key(
+        op.f("power_sensor_id_sensor_fkey"),
+        "power",
+        "sensor",
+        ["sensor_id"],
+        ["id"],
+        ondelete="CASCADE",
+    )
+
+    # Migrate Price/Market relationship to Price/Sensor relationship
+    op.drop_constraint("price_market_id_market_fkey", "price", type_="foreignkey")
+    op.drop_index("price_market_id_idx", table_name="price")
+    op.alter_column("price", "market_id", new_column_name="sensor_id")
+    op.create_index(op.f("price_sensor_id_idx"), "price", ["sensor_id"], unique=False)
+    op.create_foreign_key(
+        op.f("price_sensor_id_sensor_fkey"), "price", "sensor", ["sensor_id"], ["id"]
+    )
+
+    # Migrate Weather/WeatherSensor relationship to Weather/Sensor relationship
+    op.drop_constraint(
+        "weather_sensor_id_weather_sensor_fkey", "weather", type_="foreignkey"
+    )
+    op.create_foreign_key(
+        op.f("weather_sensor_id_sensor_fkey"),
+        "weather",
+        "sensor",
+        ["sensor_id"],
+        ["id"],
+    )
+
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
Why do we keep the original code around?

Reply:
To support your review. I'll remove it now.